assets/src/scripts/export.js

```diff
@@ -97,15 +97,14 @@ jQuery( function ( $ ) {
 
 	/* JSON Cookie. Remember to keep key/values short because a cookie has max 4096 bytes */
 	let json_cookie_key = 'pb_export';
-	let json_cookie = Cookies.getJSON( json_cookie_key );
-	if ( typeof json_cookie === 'undefined' ) {
-		json_cookie = {};
-	}
+	let json_cookie = Cookies.get( json_cookie_key );
+	json_cookie = typeof json_cookie === 'undefined' ? {} : JSON.parse( Cookies.get( json_cookie_key ) );
```

Comment:
The cookie could be empty. So if we call directly `JSON.pase( Cookies.get(...) )` it could thrown an error. That's because we get the cookie first to make sure it is defined.